### DISCUSSION TITLE
Tutorials for the relocated `ReinforcementLearningZoo` algorithms

### DISCUSSION THREAD

**Discussion by 00krishna** [JuliaReinforcementLearning/ReinforcementLearning.jl#399] [Q&A]
I was reading the RL.jl docs and it indicates that a bunch of the most recent algorithms like DQN, etc., are in the [`ReinforcementLearningZoo` repository](https://github.com/JuliaReinforcementLearning/ReinforcementLearningZoo.jl) . However, when I went to that github link, Github indicates that the content of that repo is archived and that the material has moved over to the `ReinforcementLearning.jl` repo. However, I have not really seen a simple DQN or such tutorial, to understand how to setup that problem in the package. Is there a good example of demo to look at, to understand how to use the deep learning methods in the RL.jl package?

**Comment by findmyway** [comment]
You can take a look at https://juliareinforcementlearning.org/docs/experiments/#DQN . I'll update the link.
